csp_billing_adapter/bill_utils.py

```diff
@@ -421,15 +451,16 @@ def process_metering(
 
         billed_dimensions = get_billing_dimensions(
             config,
-            billable_usage
+            billable_usage,
+            cache.get('status', {})
```

Comment:
Wondering if maybe the name of the `cache` entry should be more than a simple `status`, in case we ever wanted to add another type of status info to the cache in the future... E.g. `billing_status` makes it clear what the status is info is related to...